.env.example

```
@@ -19,8 +19,8 @@ LAB_MCP_HTTP_HOST=127.0.0.1
 LAB_MCP_HTTP_PORT=8400
 # Bearer token required on every HTTP request. Generate with: openssl rand -hex 32
 LAB_MCP_HTTP_TOKEN=
-# CORS allow-list, comma-separated origins. Empty = no CORS headers.
-LAB_MCP_HTTP_CORS_ORIGINS=
+# CORS allow-list, comma-separated origins. Empty = loopback-only defaults.
+LAB_CORS_ORIGINS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAB_CORS_ORIGINS key should go before the LAB_MCP_HTTP_HOST key

(UnorderedKey)